packages/belos/tpetra/test/MultiMatrixSolve/CMakeLists.txt

```diff
@@ -5,3 +5,21 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   COMM serial mpi
   STANDARD_PASS_OUTPUT
 )
+ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Triutils)
+IF (${PACKAGE_NAME}_ENABLE_Triutils)
+
+  TRIBITS_ADD_EXECUTABLE_AND_TEST(
+    Tpetra_pseudo_gmres_multi_hb
+    SOURCES test_pseudo_gmres_multi_hb.cpp BelosLinearMultiShiftProblem.hpp
+    COMM serial mpi
+    ARGS "--verbose --filename=orsirr1.hb"
+    STANDARD_PASS_OUTPUT
+    )
+
+  TRIBITS_COPY_FILES_TO_BINARY_DIR(Tpetra_CopyTestGmresMultiFiles
+    SOURCE_DIR ${Belos_SOURCE_DIR}/tpetra/example/BlockGmres
+    SOURCE_FILES orsirr1.hb
+    EXEDEPS Tpetra_pseudo_gmres_multi_hb
+  )
+
+ENDIF(${PACKAGE_NAME}_ENABLE_Triutils)
```

Comment:
Please use 'ENDIF()'. This can cause build errors, see https://github.com/trilinos/Trilinos/issues/12333

Reply:
Fixed

Reply:
Regarding #12333 , the change here is not sufficient to resolve #12333 , just preventative of more issues